cfme/fixtures/pytest_selenium.py

```diff
@@ -49,6 +49,24 @@
 class_selector = re.compile(r"^(?:[a-zA-Z][a-zA-Z0-9]*)?(?:[#.][a-zA-Z0-9_-]+)+$")
 
 
+# Monkeypatching WebElement
+_old__repr__ = WebElement.__repr__
+
+
+def __repr__(self):
+    if hasattr(self, "_source_locator"):
+        this, parent = self._source_locator
```

Comment:
I'm thinking a  `try/except AttributeError` might be a tiny bit cleaner (`_source_locator` is only looked up once), but feel free to ignore this comment if you want. :)